include/zephyr/drivers/mm/mm_drv_bank.h

```diff
@@ -32,7 +34,9 @@ struct mem_drv_bank {
  *
  * The driver may wish to track various information about the memory banks
  * that it uses. This routine will initialize a generic structure containing
- * that information.
+ * that information. Since at the power on all memory banks are switched on
+ * it will start with all pages mapped. In next phase of driver initialization
+ * unused pages will be unmapped.
```

Comment:
Which part of the driver initialization unused pages are unmapped ? I saw this being doing only in the test. This last sentence seems to be more related with the test or how you are using this driver than with the driver behavior itself.